<div class="row wrapper border-bottom white-bg page-heading">
    <div class="col-lg-10">
        <h2>FASTA/FASTQ format</h2>
        <ol class="breadcrumb">
            <li>
                <a href="index.html">Home</a>
            </li>
            <li>
                <a>Guides</a>
            </li>
            <li class="active">
                <strong>FASTA/FASTQ format</strong>
            </li>
        </ol>
    </div>
    <div class="col-lg-2">

    </div>
</div>
<div class="wrapper wrapper-content  animated fadeInRight article">
    <div class="row">
        <div class="col-lg-10 col-lg-offset-1">
            <div class="ibox">
                <div class="ibox-content">
                    <div class="text-center article-title">
                        <span class="text-muted"><i class="fa fa-clock-o"></i> 16th June 2016</span>
                        <h1>
                            FASTA/FASTQ format
                        </h1>
                    </div>
                    <p>
                        One of the first things you face as a newbie bioinformatician are files in FASTA/FASTQ format.
                        They are a de facto standard for storing raw input data in bioinformatics and are widely used
                        across the industry.
                        So, what exactly do these files look like?
                    </p>
                    <p>
                        First, let's discuss where FASTA/FASTQ files come from and what kind of information they contain.
                        FASTA/FASTQ files are generated by sequencing devices (<i>Illumina MiSeq/HiSeq, Roche, 
                            Ion Proton etc.</i>) that take DNA/RNA genetic material as an input. These devices (
                        <b class="text-navy">sequencers</b>) differ from each other by processing throughput, cost per run and other
                        technical characteristics. Illumina is clearly the most popular sequencer brand that has nearly 90% of the
                        market.</p>
                    <p>
                        To understand FASTA format you should first grasp the basics of what is going on in the lab.
                        In simple terms, the "magical" process that is happening at the facilities of a sequencing provider
                        looks the following way:
                    <h3>1. NGS library preparation</h3>
                    <p> Lab technician orders a <b class="text-navy">library preparation kit </b>, usually from the sequencing device manufacturer.
                        This kit is used to prepare long nucleic acid chains (DNA or RNA) for high throughput sequencing process. 
                        Preparation kits vary a lot depending on many factors, but conceptually they usually
                        comprise 3 kinds of chemical compounds: <b class="text-navy">adapters</b>,  <b class="text-navy">restriction
                            enzymes </b> and <b class="text-navy">primers</b>. </p>
                    <p>
                        So, how do these chemical compounds work? At the first stage, restriction enzymes cut the long nucleotide chains
                        of DNA or RNA into pieces of randomized length. This step is necessary because high throughput sequencing 
                        assumes reading many relatively short nucleotide chains (100-150 bases) in parallel.
                    </p>
                    <p>
                        Afterwards, the adapters come into play. These are short synthesized double-stranded DNA molecules
                        with sticky ends that are attached to every DNA/RNA fragment at both ends. After the adapters are attached,
                        the fragments are  <b class="text-navy">PCR-amplified</b> (copied) to get more genetic material available
                        for further processing.</p>
                    <p> Next, the lab technician takes a special flow cell. This flow cell has lanes with surface-bound short oligos
                        that are complementary to the adapters. The DNA/RNA fragments are ligated (attached) to these oligos, each 
                        at one end (as shown on the picture).
                    </p>
                    <h3>2. Bridge amplification and clustering</h3>
                    <p>
                        The subsequent process of bridge amplification and clustering is built around the concept of iterative
                        "reconstruction" of single-stranded DNA/RNA to double-stranded chains. This is done by <b class="text-navy">                    
                            dNTP molecules.</b> dNTP (<i>deoxyribose nucleoside triphosphate</i>) is the common name of a 
                            molecule that consists of a nucleotide (A,T,C or G), three phosphate groups and deoxyribose. </p>
                    <p> This iterative process goes the following way: remember we had our segments attached at one end. 
                        So, the next thing we do is attaching them at the second end to another oligo from the flow cell to form
                        a U-shaped "bridge". Then this single-stranded bridge is "repaired" by dNTP to 
                        a double stranded bridge chain and again broken into 2 separate strands. What we get as a result is 2
                        complementary sequences identical to each other but attached at different ends to the flow cell.</p>
                    <p>
                        The process is repeated numerous times, and as a result we get "islands" (<b class="text-navy">clusters</b>)
                        of segment copies and reverse complements, where all copies are attached at one end and all complements - 
                        at the other end. This whole process is called <b class="text-navy">bridge amplification</b>.
                    </p>
                    <h3>3. Attachment of primers and sequencing</h3>
                    <p>                           
                        Primers are used to mark the parts of DNA/RNA that are going to be sequenced, as in most cases you don't
                        want to sequence the whole genome (it's very costly) but only particular parts of it.
                        Primers possess chemical structure that allows them to bind to the specific sites on the genome marking
                        these sites for further processing.</p>

                       
                        dNTP copies each chain, and then                     
                        How does Illumina know which strands are forward and which are reverse?
                        This is how the quality of the read is determined.
                    
                    The technique commonly used in molecular biology for to amplify a single copy of DNA fragment into 
                    multiple copies is PCR (polymerase chain reaction).


                    Fluorescently labeled nucleotides
                    Solid phase amplification generates copies around the template molecule.
                    Sequencing by synthesis and dNTP
                    
                    Here's a very good video from Illumina that visualizes the whole process.

                    <b class="text-navy">Flow cell</b> has lanes

                    a lab technician 
                    prepares special chips which have fragmented 
                    nucleotide sequences attached to them. These fragmented sequences are created from long DNA or RNA 
                    nucleotide chains by cutting these chains into pieces of random length. 

                    The cutting and attachment process
                    is done using special chemical compounds that are included in <b class="text-navy">library preparation kits</b>
                    and are usually supplied by the same companies that manufacture the sequencing devices.                       
                    </p>
                    <p>
                        Every chip has several millions fragments attached. The device "reads" these nucleotide sequences, nucleotide
                        by nucleotide (base by base), in parallel fashion and records the information about them in the FASTA/FASTQ file. Every record in the file is
                        called a <b class="text-navy">read</b>. Devices differ from each other by number of the reads they can
                        produce per unit of time. For example, Illumina HiSeq produces circa XXX reads per hour and can
                        read XXX nucleotides per run.
                        Measured in kilobases per run!!!
                        And cost per gigabase.

                        Library preparation kits
                        throughput per run

                        Because of limitations in DNA sequencer technology these reads are short compared to the length of
                        a genome therefore the reads must be assembled into longer contigs.[7] The data may also contain 
                        errors, caused by limitations in the DNA sequencing technique or by errors during PCR amplification.
                        DNA sequencer manufacturers use a number of different methods to detect which DNA bases are present.
                        The specific protocols applied in different sequencing platforms have an impact in the final data that
                        is generated. Therefore, comparing data quality and cost across different technologies can be 
                        a daunting task. Each manufacturer provides their own ways to inform sequencing errors and scores. 
                        However, errors and scores between different platforms cannot always be compared directly. Since 
                        these systems rely on different DNA sequencing approaches, choosing the best DNA sequencer and
                        method will typically depend on the experiment objectives and available budget.[2]

                        prone to errors when estimating the number of bases in a long string of identical nucleotides. This is referred to as a homopolymer error 
                        HiSeq data output per run is circa 600 Gb
                        Run takes several days
                        Cost per run is 6000 USD
                    </p>
                    <p>Current technology allows the scientists to read only relatively short 
                        fragments (120-150 nucleotides), so the next challenge is to assemble those small fragments back into 
                        a genome. To make this task a little bit easier, the sequencers attach extra information to every read
                        that allows to trace back its relative location, as well as to assess how sure can we be about
                        every nucleotide in every read.

                        Paired-ended sequencing produces twice the same number of reads for the same time and effort in library
                        preparation. One can detect InDels only with paired-ended, not with single-ended data.

                    <h3>1. Sequencing by synthesis</h3>
                    <h3>2. Long read sequencing</h3>
                    <h3>3. RNA-Seq technology</h3>
                    <h3>4. Paired-end sequencing</h3>
                    <h3>5. Single-end sequencing</h3>
                    <h3>6. Multiplex sequencing</h3>
                    <h3>7. Mate pair sequencing</h3>
                    <h3>8. Deep sequencing</h3>

                    DNA sequencing -- whole genome sequencing, CHiP sequencing, targeted resequencing
                    Since paired-end reads are more likely to align to a reference, the quality of the entire data set improves.
                    Paired-end DNA sequencing reads provide superior alignment across DNA regions containing repetitive sequences, 
                    and produce longer contigs for de novo sequencing by filling gaps in the consensus sequence.

                    </p>
                    <br/>
                    <h3>1. FASTA format</h3>
                    <p>
                        So, let's finally move to the FASTA format topic and what kind of information is stored in the files of 
                        this format.
                        FASTA format is a text-based format that contains information about nucleotide sequences. 
                        Every record in the file represents a sequence and consists of two lines.
                    </p>

                    <p>
                        The first line starts with '>' symbol and is basically the "name" of the sequence, its
                        unique identifier. This identifier describes the following characteristics of the sequence:
                    <ul>
                        <li>test</li>
                        <li>test</li>
                        <li>test</li>
                        <li>test</li>
                    </ul>
                    This gives you the information about 

                    Each nucleotide
                    in every sequence is represented by a single letter code (A, C, G or T).
                    Moreover, every sequence is annotated by a name                    </p>
                    <p>
                        Two driven jocks help fax my big quiz. Quick, Baz, get my woven flax jodhpurs! "Now fax quiz Jack!" my brave ghost pled. Five quacking zephyrs jolt my wax bed. Flummoxed by job, kvetching W. zaps Iraq. Cozy sphinx waves quart jug of bad milk. A very bad quack might jinx zippy fowls. Few quips galvanized the mock jury box. Quick brown dogs jump over the lazy fox. The jay, pig, fox, zebra, and my wolves quack! Blowzy red vixens fight for a quick jump. Joaquin Phoenix was gazed by MTV for luck. A wizardâ€™s job is to vex chumps quickly in fog. Watch "Jeopardy!", Alex Trebek's fun TV quiz game. Woven silk pyjamas exchanged for blue quartz. Brawny gods just
                    </p>
                    <p>
                        Brick quiz whangs jumpy veldt fox. Bright vixens jump; dozy fowl quack. Quick wafting zephyrs vex bold Jim. Quick zephyrs blow, vexing daft Jim. Sex-charged fop blew my junk TV quiz. How quickly daft jumping zebras vex.
                    </p>
                    <hr>
                </div>
            </div>
        </div>
    </div>


</div>