<div class="wrapper wrapper-content  animated fadeInRight article">
    <div class="row">
        <div class="col-lg-10 col-lg-offset-1">
            <div class="ibox">
                <div class="ibox-content">
                    <div class="text-center article-title">
                        <span class="text-muted"><i class="fa fa-clock-o"></i> 16th June 2016</span>
                        <h1>
                            FASTA/FASTQ format
                        </h1>
                    </div>
                    <p>
                        One of the first things you face as a newbie bioinformatician are files in FASTA/FASTQ format.
                        They are a de facto standard for storing raw input data in bioinformatics and are widely used
                        across the industry. So, what exactly do these files look like? Here's an example below: 

                    </p>
                    <div class="row">
                        <div class="col-lg-8">
                            <img src='img/guides/img4.png' width="100%" > 
                        </div>

                    </div>
                    <br/>
                    <br/>
                    <h3>FASTA format</h3>
                    <p>FASTA file contains information about reads that have been performed by a sequencing device.
                        You can read more about the technical details of the sequencing process 
                        <a ui-sref='guides.article4'>in this article</a>. Information about each read consists of
                        2 lines: first line with a description header and second line with the sequence of nucleotides.</p>

                    <p>
                        <b>The first line of the read record </b> starts with '>' or @ symbol and is basically the "name" of the sequence, its
                        <b class="text-navy">unique identifier</b>. In general, the format of this identifier is not restricted at all, so
                        if you create a FASTA file manually, you can name your reads whatever you like, and the file
                        will be perceived as a valid one by all kinds of bioinformatics software. However,
                        most sequencing device producers tend to encode into this line important information about the reads
                        that will allow the users to make judgments about where the reads came from and how they relate
                        to each other. For instance, for Illumina devices, the standard name line contains such
                        information as the record header, flow cell ID, lane, tile, tile coordinates and barcode.
                    </p>
                    <div class='row'><div class='col-lg-6'>
                            <p>
                                The information about the location of the read on the flow cell lets us trace back, if there have 
                                been any technical issues during the sequencing process. For instance, the flow cell could have been
                                scratched or contaminated at a particular spot of one of its lanes, or there could have been a small air
                                bubble, or an electrostatic effect - the list of potential problems goes on and on.
                                If we incorporate the information about the location of the reads, we can later use quality control tools
                                (like <b class="text-navy">FASTQC</b>, for instance) to filter out such reads (see image 1 on the right for sample FASTQC output).
                            </p>
                        </div>
                        <div class="col-lg-6">
                            <h4>Image 1. FASTQC output for quality per tile</h4>
                            <img src='img/guides/img5.png' width="100%" > 
                        </div>
                    </div>
                    <p>
                        If reads are paired-ended, the first line also contains the number of the read in the pair (1 or 2), where
                        1 stands for forward strand of the DNA/RNA, and 2 - for the reverse strand. In most cases, reads from the
                        forward and from the reverse strands are saved into 2 separate FASTA files for convenience purposes;
                        however, nothing prevents them from being stored in a single file. In general, if the reads are paired,
                        they should have the same header line except for one symbol (1 or 2 correspondingly).
                    </p>

                    <p>
                        <b>The second line of the read record</b> in a FASTA file is simply a long string that consists of A, C, G and T letters.
                        Of course, if our file contains information about RNA instead of DNA, we will see U-s instead of T-s in it. Also, 
                        if a sequencing device is unable to read a nucleotide at a particular place with sufficient confidence,
                        it usually puts a letter N for it. Most bioinformatics software is customized for handling such cases.
                    </p>
                    <br/>
                    <br/>
                    <h3>FASTQ format</h3>
                    <p>
                        FASTQ format is very similar to FASTA format, except for the fact that has 2 additional lines per read. Lines
                        number 1 and 2 have exactly the same format both in FASTA and FASTQ files.
                    </p>
                    <p>
                        <b>The third line of the read record</b> in a FASTQ file is simply a plus character that acts as a divider, for
                        convenience purposes.
                    </p>
                    <p>
                        <b>The fourth line of the read record</b> is a very important one. It has the same length as line 2 and
                        provides us information about the sequencing quality of each of the nucleotides in the read. The sequencing 
                        quality of every nucleotide is measured by number from 1 to 128, and this number is encoded using 
                        standard ASCII-table. The character '!' represents the lowest quality while '~' is the highest.
                    </p>
                    <p>
                        To make it more clear, let's consider and example. Here is a list of 128 ASCII characters: <br/>
                        <br/>
                        !"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~ <br/>
                        <br/>
                        Suppose the first nucleotide in our read is read with the best quality possible, and we are 100% sure that it 
                        is adenine. Then, line 2 of the record in the FASTQ file will begin with 'A' character, and line 4 will 
                        begin with '~' character.
                    </p>
                    <p>
                        So, how exactly do we map the quality of the reads to 128 symbols? And, overall, how do we measure quality?
                        The technique is based on <b class="text-navy">Phred quality score</b>. This exact formula slightly varies
                        across different versions of Illumina platform. Starting from Illumina 1.8, the formula used is
                        <b>-10log<sub>10</sub>(p)+33</b>, where p is the probability that the corresponding nucleotide base has been
                        read incorrectly. So, for best quality possible the probability that the read is incorrect is smaller than 
                        0.0000001%.
                    </p>
                    <p>
                        Another point worth mentioning is that the quality of the nucleotides read usually decreases towards the
                        end of the read. This is due simply to the technical features of the sequencing process. Hence, a common
                        technique is to cut the ends of the reads if the quality falls below certain threshold. Again, FASTQC 
                        provides tools for visual analysis of this effect (see image 2 for comparison of good and bad Illumina data).                      
                    </p>
                    <div class="row">
                        <div class="col-lg-12">
                            <h4>Image 2. Good (left) and bad (right) Illumina data </h4>
                            <img src='img/guides/img6.png' width="100%">
                        </div>
                    </div>
                    <hr>
                </div>
            </div>
        </div>
    </div>


</div>