<div class="wrapper wrapper-content  animated fadeInRight article">
    <div class="row">
        <div class="col-lg-10 col-lg-offset-1">
            <div class="ibox">
                <div class="ibox-content">
                    <div class="text-center article-title">
                        <span class="text-muted"><i class="fa fa-clock-o"></i> 16th June 2016</span>
                        <h1>
                            Genomics 101: what is going on in the lab?
                        </h1>
                    </div>
                    <p>
                        To understand bioinformatics better, you should first familiarize yourself at a high level
                        with the process that is going on in the lab. Basically, the data is generated by sequencing devices 
                        (<i>Illumina MiSeq/HiSeq, Roche, Ion Proton etc.</i>) that take DNA/RNA genetic material as an input.
                        These devices (<b class="text-navy">sequencers</b>) differ from each other in processing throughput, cost per run and other
                        technical characteristics. Illumina is clearly the most popular sequencer brand that has nearly 90% of the
                        market. Sequencing providers operate such devices in their labs, and the "magic" that is happening at their
                        lab facilities looks the following way:</p>
                    <h3>1. NGS library preparation</h3>
                    <div class="row">
                        <div class='col-lg-8'>
                            <p> Lab technician orders a <b class="text-navy">library preparation kit </b>, usually from the sequencing device manufacturer.
                                This kit is used to prepare long nucleic acid chains (DNA or RNA) for high throughput sequencing process. 
                                Preparation kits vary a lot depending on many factors, but conceptually they usually
                                comprise 3 kinds of chemical compounds: <b class="text-navy">adapters</b>,  <b class="text-navy">restriction
                                    enzymes </b> and <b class="text-navy">primers</b>. </p>
                            <p>
                                So, how do these chemical compounds work? At the first stage, restriction enzymes cut the long nucleotide chains
                                of DNA or RNA into pieces of randomized length. This step is necessary because high throughput sequencing 
                                assumes reading many relatively short nucleotide chains (100-150 bases) in parallel, while the length of human genome
                                is over 3 billion nucleotides.
                            </p>
                            <p>
                                Afterwards, the adapters come into play. These are short synthesized double-stranded nucleotide chains
                                with sticky ends that are attached to every DNA/RNA fragment at both ends (see image 1). Two different kinds
                                of adapters are used simultaneously to mark the start and the end of every fragment.
                            </p>
                        </div>
                        <div class="col-lg-4">
                            <h4>Image 1. Adapter ligation</h4>
                            <img src='img/guides/img1.png' width="100%" > 
                        </div>
                    </div>
                    <p> Now, we need to create multiple copies of our fragments to ensure that when we will later read them with a 
                        sequencer, we wouldn't make mistakes. So, after the adapters have been attached,
                        the fragments are  <b class="text-navy">PCR-amplified</b> (copied) to get more genetic material available
                        for further processing. During PCR-amplification double-stranded nucleotide chains are broken into single-stranded ones
                        (this process is called <b class="text-navy">denaturation </b> and happens at approximately 94-96 degrees Celsius).
                        Then each of the strands is "repaired" back by <b class="text-navy">DNA-polymerase</b> to its double-stranded version (remember, A sticks
                        to T and C sticks to G). The process is repeated again and again, so as a result we get multiple copies of 
                        the same nucleotide chain. </p>
                    <p>One of the key points here is that PCR-amplification can't be done without primers. DNA-polymerase can't just
                        begin "repairing" the single-stranded chains at a random place, it needs a "hook" to know where to start. 
                        These "hooks" are formed by primers that we attach to our single-stranded chains. Primers are short (about 20 bases)
                        nucleic acid sequences complementary to certain parts of DNA/RNA fragments. DNA-polymerase finds a primer and starts
                        the single-strand reparation process from it onwards, forming a double-stranded chain from a single-stranded one.
                    </p>
                    <div class="row">
                        <div class='col-lg-4'>
                            <h4>Image 2. Attachment to the flow cell</h4>
                            <img src='img/guides/img2.png' width="100%">
                        </div>
                        <div class='col-lg-8'>
                            <p> Now, say we've completed the PCR-amplification and eventually have enough genetic material available for
                                further processing. Next, the lab technician takes a special flow cell with lanes on it. 
                                The lanes have surface-bound short oligonucleotides (oligos) that are complementary to the adapters we attached. 
                                The DNA/RNA fragments are ligated (attached) to these oligos, each at one end.
                            </p>
                            <p>
                                How exactly does the ligation process happen? A single-stranded fragment sticks to the oligo, as they are
                                complementary to each other, and the latter is "glued" to the flow cell surface. Then, the oligo works
                                basically as a primer for DNA-polymerase, as the polymerase rebuilds the single-stranded chain to a double-
                                stranded one starting from the oligo. Then, we heat the flow cell to denaturate the chains and wash away
                                the single-strands not attached to the surface. Eventually we get a "forest" of single-stranded chains 
                                safely attached at one end to the surface (as shown on the image 2).
                            </p>
                        </div>
                    </div>
                    <br/>
                    <h3>2. Bridge amplification and clustering</h3>
                    <p>
                        Next thing we do is, again, iteratively multiplying the number of "trees" in our "forest" to make it easier
                        for a sequencing device to read the nucleotides. This is done via a process that is very similar to
                        PCR-amplification and that is called <b class="text-navy">bridge amplification</b>.</p>
                    <p>    
                        Bridge amplification process follows essentially the same cycle (<i>attach primer -> apply DNA polymerase ->
                            denaturate double-stranded chains into single-stranded ones -> repeat</i>). The only difference
                        here is that our initial sequences are now attached at one of the ends. The trick 
                        is that we use the free attached oligos left on the lanes as our primers. In other words, 
                        we form U-shaped bridges from the single-stranded nucleotide chains, and then run the amplification
                        process on them. As a result of each iteration, we get 2 complementary sequences identical to 
                        each other but attached at different ends to the flow cell. </p>
                    <p>
                        The process is repeated numerous times, and as a result we get "islands" (<b class="text-navy">clusters</b>)
                        of segment copies and their reverse complements, where all copies are attached at one end and all complements - 
                        at the other end. Now, we are eventually ready to sequence the data on our flow cell.
                    </p>
                    <div class="row">
                        <div class="col-lg-12">
                            <h4>Image 3. Bridge amplification process</h4>
                            <img src='img/guides/img3.png' width="100%">
                        </div>
                    </div>
                    <h3>3. Sequencing</h3>
                    <p>                           
                        Sequencing process is done iteratively, nucleotide by nucleotide, but on all attached fragments in parallel.
                        First, again, we attach the primers to our single-stranded sequences, to mark the starting point for
                        sequence reconstruction. Next, instead of DNA-polymerase, we spray a mix of four 
                        types of reversible terminator bases <b class="text-navy">(RT-bases)</b> on the flow cell.
                        Each RT-base consists of one of four nucleotides plus a fluorescent dye.
                        Nucleotides in the RT-bases attach to our single-stranded chains at the spots marked by primers
                        and fluoresce. A camera takes an image of the surface, determining which fluorescent nucleotide
                        has been attached to each fragment, thereby "reading" one nucleotide per fragment.</p>
                    <p>    
                        An important point about RT-bases is the following: they are able to
                        attach to the nucleotide chains <i>only one base at a time</i>, as they contain a terminal 3' blocker.
                        But then, after the camera has taken the image, they can be treated chemically to remove the blocker 
                        and the dye and become a binding point for attachment of the next RT-base nucleotide. In this way, the 
                        sequencing device can read the nucleotide chains base-by-base, taking an image and then washing away 
                        the blockers and the dye at each iteration.
                    </p>    
                    <p>
                        Technical limitation on the number of iterations is 100-150 launches. In other words, an Illumina 
                        sequencing device can read the fragments of 100-150 nucleotides long. These processed fragments are recorded
                        in the output file and called <b class="text-navy">reads</b>. The number of fragments read in parallel
                        at every "shot" of the camera is several million fragments, so the processing speed of a sequencing device
                        is pretty fast (200 GB of output data can be generated in one day by a HiSeq device).  
                    </p>    
                    <p>
                        What kind of extra information does a sequencing device record apart from the nucleotides themselves?
                        Lots of it. First of all, it takes notes of the fragment location on the flow cell/lane to trace back the
                        local impurities/contamination, if any. Second, as many of the reads are exact copies of each other, the
                        device validates its own processing quality by comparing the reads across every cluster. Finally,
                        the sequencer stores the information about whether the reads are <b class="text-navy">paired-ended </b>
                        which means that they are taken from the same cluster, but from a fragment and its reverse complement.
                        All this information is also stored in the output files (FASTA/FASTQ) in different ways, together with the reads themselves.
                    </p>
                    <br/>
                    <hr>
                </div>
            </div>
        </div>
    </div>


</div>